engine/src/main/java/org/terasology/logic/ai/AICommands.java

```diff
@@ -25,26 +25,38 @@
 import com.google.common.collect.Iterables;
 
 /**
+ * Artificial Intelligence command
  */
 @RegisterSystem
 public class AICommands extends BaseComponentSystem {
     @In
     private EntityManager entityManager;
 
+    /**
+     * Lists all AIs in the world
+     * @return String string containing number of simple AIs and hierarchical AIs
+     */
     @Command(runOnServer = true, shortDescription = "Count all AIs in the world")
     public String countAI() {
         int simpleAIs = Iterables.size(entityManager.getEntitiesWith(SimpleAIComponent.class));
         int hierarchical = Iterables.size(entityManager.getEntitiesWith(HierarchicalAIComponent.class));
         return "Simple AIs: " + simpleAIs + ", Hierarchical AIs: " + hierarchical;
     }
 
+    /**
+     * Destroys all AIs in the world
+     * @return String string containing number of simple AIs and hierarchical AIs
+     */
     @Command(runOnServer = true, shortDescription = "Destroys all AIs in the world")
     public String destroyAI() {
         int simpleAI = 0;
+        // go through all simple AIs and destroy reference
```

Comment:
For once this comment is probably not needed - the code itself is pretty clear (and technically it is destroying the entities that have the component attached - not references. EntityRef is a reference to an entity)